### PR TITLE
First pages for RHPDS instructions

### PR DESCRIPTION
**Pull request by ge0ffrey**
@yurloc Please link the https://www.optaplanner.org/learn/rhpds/optaweb-vehicle-routing.html url from your description.

### CODE REVIEW

File: learn/rhpds/optaweb-employee-rostering.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+= RHPDS instructions for OptaWeb Vehicle Routing
```
Comment (ge0ffrey) [inline learn/rhpds/optaweb-employee-rostering.adoc]: ok, fixing this...

File: learn/rhpds/index.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+= RHPDS instructions for OptaPlanner demo's
```
Comment (yurloc) [inline learn/rhpds/index.adoc]:
```suggestion
= RHPDS instructions for OptaPlanner demos
```

File: learn/rhpds/index.adoc [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+:awestruct-description: How to demo OptaPlanner with RHPDS
+:awestruct-layout: normalBase
+:awestruct-priority: 0.1
+:showtitle:
+
+== Visit RHPDS
+
+https://rhpds.redhat.com/[Visit RHPDS]
+
+== Available demo's on RHPDS
```
Comment (yurloc) [inline learn/rhpds/index.adoc]:
```suggestion
== Available demos on RHPDS
```

**Comment by ge0ffrey** [comment]
Applied all Jiri's suggestions, thanks Jiri

**Pull request merged**